lib/beaker/hypervisor/docker.rb

```diff
@@ -102,7 +113,7 @@ def provision
         fix_ssh(container) if @options[:provision] == false
 
         @logger.debug("Starting container #{container.id}")
-        container.start({"PublishAllPorts" => true, "Privileged" => true})
```

Comment:
looking at their [API](https://docs.docker.com/engine/api/v1.30/#operation/ContainerStart) they dont have any options that are provided with `start` (i understand that the code still support taking options on `start`). but looks like [creating](https://docs.docker.com/engine/api/v1.30/#operation/ContainerCreate) container is where the options are passed

i also tried a combination of keeping the options provided with `start` as is and passing other options with `create` but all the options passed with `create` is being overwritten 😞